lib/ContentTypeParser.js

```diff
@@ -42,7 +42,7 @@ ContentTypeParser.prototype.run = function (contentType, handler, handle, params
       const reply = new handle.Reply(req, res, handle)
       return reply.send(body)
     }
-    handler(handle, params, req, res, body, query)
+    handler(handle, params, req, res, body, query, req.headers)
```

Comment:
`req.headers` here is not needed, you can directly access it as we are doing for `req.log` at line 84.